### DISCUSSION TITLE
When only using a Node target, how can I remove CSS dependencies?

### DISCUSSION THREAD

**Discussion by zapplebee** [parcel-bundler/parcel#6907] [Q&A]
I love Parcel for quickly setting up building Node targets with Babel.

Because I am building Node apps, I have no need for the CSS dependencies, (postcss etc).

Is there a way for me to exclude these deps?

**Comment by oriSomething** [comment]
I would add, that even without `node`, since, sometimes I need a library to consume the CSS differently
